spyder/app/mainwindow.py

```diff
@@ -1316,6 +1319,17 @@ def setup_menus(self):
                         lambda menu=menu: set_menu_icons(menu, False))
                     menu.aboutToShow.connect(self.hide_options_menus)
 
+    def update_lsp_logs(self):
+        """Create an action for each lsp log file."""
+        lsp_logs = []
+        regex = re.compile(r'.*_.*_(\d+)[.]log')
+        files = glob.glob(osp.join(get_conf_path('lsp_logs'), '*.log'))
+        for f in files:
+            action = create_action(self, f, triggered=self.editor.load)
+            action.setData(to_qvariant(f))

```

Comment:
@steff456 I don't think we need to use `to_qvariant`

`action.setData(to_qvariant(f))` -> `action.setData(f)`

Could you check it works?